config/environments/test.rb

```diff
@@ -0,0 +1,60 @@
+require "active_support/core_ext/integer/time"
+
```

Comment:
- [ ] [OPTIONA] It would be great if you can add more tests.

Reply:
Thanks for the suggestion! I'll try to improve it in the next few days!